docs/pages/distribution/hosting-your-app.md

```diff
@@ -2,6 +2,8 @@
 title: Hosting Updates on Your Servers
 ---
 
+import { InlineCode } from '~/components/base/code';
+
 Normally, when updates are enabled, your app will fetch updates comprising JavaScript bundles and assets from Expo’s CDN. However, there will be situations when you will want to host your JS bundles and assets on your own servers. For example, updates are slow or unusable in countries that have blocked Expo’s CDN providers on AWS and Google Cloud. In these cases, you can host your updates on your own servers to better suit your use cases.
```

Comment:
we don't support self-hosted managed apps on eas build at the moment, users would have to do this with a bare app